### PR TITLE
Added methods to rotation and vector3 and removed some methods from car

### PR DESCRIPTION
**Pull request by LHolten**
Please review these changes before merging :)

### CODE REVIEW

File: dataParse/dataParse/dataParse.py [hunk 1]
```diff
@@ -7,10 +7,7 @@
 
 
 class PythonExample(BaseAgent):
-
-    def initialize_agent(self):
-        # This runs once before the bot starts up
-        self.controller_state = SimpleControllerState()
+    controller_state = SimpleControllerState()
```
Comment (ard1998) [inline dataParse/dataParse/dataParse.py]: dont we need to put this in initialize_agent()?

File: dataParse/dataParse/dataParse.py [hunk 2]
```diff
@@ -107,36 +101,13 @@ def __init__(self, car, index):
         self.team = car.team
         self.boost = car.boost
```
Comment (ard1998) [inline dataParse/dataParse/dataParse.py]: il need to remove these 2 lines, whey were still there from me testeing some rlbot stuff.

**Comment by ard1998** [comment]
everything seems to work on installation. thanks for the code improvement @LHolten

File: dataParse/dataParse/dataParse.py (hunk 2 above)
Comment (LHolten) [reply dataParse/dataParse/dataParse.py]: wait, why do you have to remove those?

Comment (ard1998) [reply dataParse/dataParse/dataParse.py]: forgot they were used 😂

**Pull request merged**